jobs/aws/eks-distro/announcement-postsubmits.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+postsubmits:
+  aws/eks-distro:
+  - name: announcement-postsubmit
+    always_run: false
+    run_if_changed: "release/.*/release-announcement.txt"
```

Comment:
I know this is what the proposed path in my open PR, but we may want to change where this is located. I'll follow up with you about this. It's a two way door, so this shouldn't block us